iib/workers/tasks/opm_operations.py

```diff
@@ -499,6 +499,34 @@ def generate_cache_locally(base_dir: str, fbc_dir: str, local_cache_path: str) -
         raise IIBError(error_msg)
 
 
+def extract_fbc_fragment_config(temp_dir: str, fbc_fragment: str) -> None:
+    """
+    extract catalog.json file from the from fbc_fragment image
+    return fbc_fragment path
+    """
+    log.info("Extracting the fbc fragment %s", fbc_fragment)
+
+    # TODO get fbc path from conf?
+    # store the fbc_fragment at /tmp/iib-**/fbc-fragment
+    fbc_fragment_path = os.path.join(temp_dir, 'fbc_fragment')
+    os.makedirs(os.path.dirname(fbc_fragment_path), exist_ok=True)
+
+    from iib.workers.tasks.build import _copy_files_from_image
+
+    # Verify from opm team if the dir path will be available in a label
+    # Verify this will this be catalog or configs?
+    _copy_files_from_image(fbc_fragment,'/configs', fbc_fragment_path)
```

Comment:
To be persistent and not use hard-coded string we might use:`fbc_dir_path = get_image_label(fbc_fragment, 'operators.operatorframework.io.index.configs.v1')`  
If it is in the config dir, if not I would vote for agreeing on setting label as we do for configs instead of hard-coding path.

Reply:
since we are still not sure if the path will be set in a label, I added a worker config for that